rust-toolchain

```diff
@@ -7,14 +7,13 @@
 #   supported and used in CI.
 
 [toolchain]
-channel = "1.52.1"
+channel = "1.55.0"
```

Comment:
Given that there is quite a few (mechanical) changes. @jhugman, @travis79  -- this is where the main rust version bump actually comes from